kinoshita/laravel-study/miniblog/app/Http/Controllers/PostController.php

```diff
@@ -68,7 +69,12 @@ public function store(PostRequest $request)
      */
     public function show(Request $request, Post $post)
     {
-        //
+        $comments = Comment::with('user')->where('post_id', $post->id)->get();
```

Comment:
リレーション使って欲しいです。(withがダメかもしれない。その場合代わりにloadでいけるはずです。)

```suggestion
        $comments = $post->comments()->with('user')->get();
```

Reply:
動的プロパティとリレーションオブジェクトの違いがよくわかってなかったので、調べてsuggested changeの内容で修正しました
https://qiita.com/179Bell/items/7b4f991816f63946e738